### PR TITLE
Toolbar Refactor

### PR DESCRIPTION
**Pull request by xnkevinnguyen**
# Description:
Reduce the number of components created such as temperature component by using a genericSliderComponent. Each devices declares it's modal content and it's properties.

This is a refactoring change.

List of components that were removed and replaced with a reusable alternative: 
-TemperatureSensorBar.tsx
-MotionSensorBar.tsx
-Accelerometer.tsx
-LightSensorBar.tsx

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by sagarmanchanda — APPROVED** [review]
+1 for the refactor. Ship it! 💯

File: src/view/components/toolbar/clue/ClueSensorProperties.tsx [hunk 1]
```diff
@@ -146,3 +146,54 @@ export const CLUE_PRESSURE_PROPERTIES: ISensorProps = {
...
+    maxLabel: "Front",
+    maxValue: 1023,
+    minLabel: "Back",
+    minValue: -1023,
+    type: SENSOR_LIST.MOTION_Y,
+};
+
+const MOTION_SLIDER_PROPS_Z: ISliderProps = {
+    axisLabel: "Z",
+    maxLabel: "Down",
```
Comment (vandyliu) [inline src/view/components/toolbar/clue/ClueSensorProperties.tsx:169]: Isn't up usually positive? I'm not an expert on z-axis stuff though.
Edit: After testing, it looks like this is correct with the actual device

File: src/view/components/toolbar/microbit/MicrobitSensorProperties.tsx [hunk 2]
```diff
@@ -1,65 +1,69 @@
...
+    maxLabel: "Front",
+    maxValue: 1023,
+    minLabel: "Back",
+    minValue: -1023,
+    type: SENSOR_LIST.MOTION_Y,
+};
+
+const MOTION_SLIDER_PROPS_Z: ISliderProps = {
+    axisLabel: "Z",
+    maxLabel: "Down",
```
Comment (vandyliu) [inline src/view/components/toolbar/microbit/MicrobitSensorProperties.tsx:39]: Same as above
Edit: Nvm

**Review by vandyliu — APPROVED** [review]
Cool stuff

**Pull request merged**